### ISSUE TITLE
How to map NS430 commands in FC3 modules?

### ISSUE THREAD

**Issue by Nick-Thain** [DCS-Skunkworks/DCSFlightpanels#109]
Gudday,
I'm one of those weirdos who enjoys playing with the NS430 in DCS, and I noticed that I can't add the NS430 to a basic key emulator profile, which is how I thought I could get around the FC3 modules not being clickable.. How can I create a profile that will allow me to use the flight panels to manipulate the NS430 in FC3 modules?

Thanks!
Nick

**Comment by jdahlblom** [comment]
Regarding NS430 and FC3. I really do not know anything about NS430 or FC3. I just implemented the DCS-BIOS part of NS430. I guess it would be better to ask on Discord of users experiences and how they use it. BadCrC for example. He might have mentioned it in his video here : https://www.youtube.com/watch?v=lgTJa-NUnM8
But I think FC3 is the limiting factor here, not sure though.

**Comment by WarLord211** [comment]
i think Nich means that it would nice if NS430 can activated also in keyemulator mod.
atm its grayed out.

**Comment by Nick-Thain** [comment]
To offer further clarification;
 - In the early days of DCSFP, NS430 was offered as it's own DCS-BIOS module,
 - After a request from a user in the forum, NS430 was removed as it's own module and integrated as an option for other modules, so that you could control the NS430 and your choice of DCS-BIOS aircraft.
 - I know that FC3 aircraft don't have DCS-BIOS modules as they don't have clickable cockpits.
 - I'd like to control the NS430 in FC3 aircraft.
 - I think the only way I could do this would be if you re-added the NS430 module as a standalone DCS-BIOS module.

Would it be possible to re-add a standalone NS430 module while keeping the NS430 also integrated to the other modules?

**Comment by jdahlblom** [comment]
OK, I understand. I will look into this. I didn't know FC3 as such had support for NS430.

**Comment by Nick-Thain** [comment]
Thanks Jerker!
I was thinking it wouldn't be too difficult given that you initially had
the NS430 as a stand alone module in DCSFP.. if I'm wrong on that point, I
can live without it ;) . I just thought if it was pretty straightforward,
and you didn't mind re-adding it, it would make interacting with the NS430
in FC3 modules SOOOOO much easier with the inner/outer knobs of the radio
panel :).
Thanks again mate,
Nick

**Comment by jdahlblom** [comment]
@Nick-Thain, could you DM on our Discord?

**Comment by Nick-Thain** [comment]
I'm there now, I assume I DM ArturDCS?

**Comment by jdahlblom** [comment]
Seems like it is not possible at the moment. For some reason there aren't any devices in the lua environment corresponding to the devices that *are there* whilst having a non-FC3 module.

**Comment by Nick-Thain** [comment]
Gudday mate,
Yeah I'm finding the same thing here, I'm just grateful that the NS430
works with the other DCS-BIOS modules now! Really appreciate you taking the
time to take a deeper look at it for me mate.

In the interim, I found a great mod on the DCS User Files that fixes the
rotary dial issue where each key press used to equal several 'clicks' of a
rotary dial; with the mod, each key press turns exactly one 'step'/click,
so I've now mapped that function to an MFD button pair.

Thanks again!
Nick
